deebot_client/capabilities.py

```diff
@@ -122,7 +122,7 @@ class CapabilityClean:
     action: CapabilityCleanAction
     continuous: CapabilitySetEnable[ContinuousCleaningEvent]
     count: CapabilitySet[CleanCountEvent, int] | None = None
-    log: CapabilityEvent[CleanLogEvent]
+    log: CapabilityEvent[CleanLogEvent] | None = None
```

Comment:
Why must these be optional?

Reply:
Because the T10 is not use the current implemented log version, for what the issue #350 is for. 
In the p95mgv.py definition I not include the log part.

Reply:
Please revert this change as otherwise the HA component needs an update too and we merge it after #350 is merged

Reply:
We can leave it